app/services/network_topology_service.rb

```diff
@@ -94,7 +94,7 @@ def entity_status(entity)
     when Vm
       entity.power_state.capitalize
     when ManageIQ::Providers::NetworkManager
-      entity.authentications.empty? ? 'Unknown' : entity.authentications.first.status.capitalize
+      entity.authentications.blank? ? 'Unknown' : entity.authentications.first.status.try(:capitalize)
```

Comment:
wait, `authentications.nil?` or `authentications.first.nil?`?

If the latter, then my issue is `entity.authentications.blank?` will fetch all authentications for the `blank?` -- the blank is not so nice